### PR TITLE
Added examples in documenation for add del delete repr insert sizeof high low

### PR DESCRIPTION
**Pull request by TheAnonymous**

### CODE REVIEW

File: lib/system.nim [hunk 1]
```diff
@@ -705,12 +719,22 @@ template excl*[T](s: var set[T], flags: set[T]) =
...
+  ##
+  ## .. code-block:: nim
+  ##  var i = {1,2,3,4}
+  ##  card(i) #=> 4
 
 proc ord*[T](x: T): int {.magic: "Ord", noSideEffect.}
   ## returns the internal int value of an ordinal value ``x``.
+  ##
+  ## .. code-block:: nim
+  ##  card('A') #=> 65
```
Comment (dom96) [inline lib/system.nim]: ``card``? Shouldn't this be ``ord``?

File: lib/system.nim [hunk 2]
```diff
@@ -1329,6 +1357,10 @@ proc shallowCopy*[T](x: var T, y: T) {.noSideEffect, magic: "ShallowCopy".}
 proc del*[T](x: var seq[T], i: Natural) {.noSideEffect.} =
   ## deletes the item at index `i` by putting ``x[high(x)]`` into position `i`.
   ## This is an O(1) operation.
+  ##
+  ## .. code-block:: nim
+  ##  var i = @[1,2,3,4,5]
+  ##  i.del(2) #=> @[1, 2, 5, 4]
```
Comment (dom96) [inline lib/system.nim]: This looks incorrect

File: lib/system.nim [hunk 3]
```diff
@@ -1336,6 +1368,10 @@ proc del*[T](x: var seq[T], i: Natural) {.noSideEffect.} =
 proc delete*[T](x: var seq[T], i: Natural) {.noSideEffect.} =
   ## deletes the item at index `i` by moving ``x[i+1..]`` by one position.
   ## This is an O(n) operation.
+  ##
+  ## .. code-block:: nim
+  ##  var i = @[1,2,3,4,5]
+  ##  i.del(2) #=> @[1, 2, 4, 5]
```
Comment (dom96) [inline lib/system.nim]: This uses ``del`` not ``delete``.

File: lib/system.nim (hunk 2 above)
Comment (TheAnonymous) [inline lib/system.nim]: Tested the code with
```
var i = @[1,2,3,4,5]
echo(i) #=> @[1, 2, 3, 4, 5]
i.del(2)
echo(i) #=>  @[1, 2, 5, 4]
```

Comment (dom96) [inline lib/system.nim]: oh yeah, it's by-index, my bad.

**Pull request merged**